src/utils.js

```diff
@@ -145,13 +145,17 @@ export function saveUser(userId) {
       }
 
       slack.web.users.info(userId)
-        .then((userInfo) => {
+        .then(async (userInfo) => {
```

Comment:
Please don't mix promises and async/await in the same context. Here you can return `listRecords` that is a promise and continue with `then` on it.